generators/init/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint no-underscore-dangle: 0 */
+
+/*
+* License-Identifier: Apache-2.0
+*/
+const Generator = require('yeoman-generator');
+const simpleFabricaConfigJson = require('../../samples/fabricaConfig-1org-1channel-1chaincode');
+
+module.exports = class extends Generator {
+  async copySimpleFile() {
+    this.fs.writeJSON(
+      'fabrica-network-config.json',
```

Comment:
@dzikowski jeszcze parę pytań do dyskusji:
1. Jaką wersję Fabrica chcemy ? Teraz jest `1.4.3`. Może dac `1.4.6` ?
2. ogranizacja posiada 2 peery. chcemy tyle czy tylko 1 żeby szybo się uruchamiało?
3. podaję też config dla chaincode'u. Chcemy takie coś? Czy umożliwiamy stawianie sieci w ogóle bez chaincodeów?

Reply:
1. dałbym 1.4.6
2. to jest właśnie powiązane z raft vs solo. czy chcemy mieć środowisko bliskie produkcyjnemu, czy chcemy mieć jak najmniejsze (więc mamy też pytanie, czy chcemy `rootCaOnly`). może z tym TLS faktycznie jest problem, bo wtedy przy invoke musisz dawać certy 🤔 
3. no to też jest problematyczne, bo jeśli nie chcesz dawać żadnego wzoru chaincoda na start, to możesz albo (1) kazać użytkownikowi jako prerequisite mieć chaincode na dysku w określonym miejscu (i w określonym języku), albo (2) pobierać te wartości jako parametry. Dlatego mi się wydaje, że dobrze by było dawać ten nasz minimal JS chaincode.